### PR TITLE
Fix results uploading form to prevent from multiple accidental submissions

### LINKED ISSUE #3045: Bug- Clicking Results Upload Button
From time to time WRT members will have an issue when uploading results where each result will be uploaded multiple times.
screen shot 2018-06-24 at 12 17 39 pm
For us to fix this, we must remove the temporary results and scrambles then reupload.

I believe I have nailed down the cause to hitting the upload button multiple times before receiving a request.

I also believe that the upload button calls a post method based on the output of Network in DevTools. I think a possible fix would be to change this to a put method.

### PR DESCRIPTION
**Pull request by jonatanklosko**
Fixes #3045.
@Jambrose777 I've just deployed this to the staging, could you test it out please?

### CODE REVIEW

**Comment by Jambrose777** [comment]
I got an invalid value submitted for field 'submit' when uploading results that were perfect on production.
screen shot 2018-07-03 at 11 33 10 am

**Comment by jonatanklosko** [comment]
Right, it turns out that the backend code requires the input value to be submitted, whereas disabling it effectively prevents from this. I decided to be less fancy and just hide the button once clicked instead (I don't believe it's worth spending too much time on). Please give it another try, hopefully this time it does the job!

File: webroot/results/js/data_upload_help.js [hunk 1]
```diff
@@ -62,4 +62,8 @@ $(document).ready(function() {
   selectize_competition_field("#form-element-competitionId");
   get_comp_info(0);
   $('#form-element-competitionId').change(get_comp_info);
+  // Prevent from multiple form submissions.
```
Comment (jfly) [inline webroot/results/js/data_upload_help.js]: nit: You can drop the "from", simply "Prevent multiple form submissions" will do!

**Comment by Jambrose777** [comment]
Looks good on staging!

**Comment by jonatanklosko** [comment]
Deployed! :shipit:

**Pull request merged**